index.html

```diff
@@ -1,12 +1,19 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
-    <title>freecodecamp-git</title>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="css/style.css" rel="stylesheet">
-  </head>
-  <body>
-    <h1>Welcome to you taherallaa</h1>
-  </body>
+
+<head>
+  <title>freecodecamp-git</title>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link href="css/style.css" rel="stylesheet" />
+</head>
+
+<body>
+  <h1>Welcome to you taherallaa</h1>
+  <p>
```

Comment:
Can you remove lorem text and add meaningful text

Reply:
ok, I'll do it